### PR TITLE
content/en/tracing/setup_overview: update OpenTelemetry documentation

### PR DESCRIPTION
**Pull request by gbbr**
This updates OpenTelemetry tracing setup documentation. Please only publish after 7.35.0 is released.

### CODE REVIEW

File: content/en/tracing/setup_overview/open_standards/_index.md
Comment (mx-psi) [inline content/en/tracing/setup_overview/open_standards/_index.md:327]: Instead of removing this, let's say that metrics is beta.

File: content/en/tracing/setup_overview/open_standards/_index.md
Comment (mx-psi) [inline content/en/tracing/setup_overview/open_standards/_index.md:386]: Comment so that we remember to change to `7.35` once released.

File: content/en/tracing/setup_overview/open_standards/_index.md
Comment (jtappa) [inline content/en/tracing/setup_overview/open_standards/_index.md:338]:
```suggestion
You can also enable these with environment variables. For example, the gRPC endpoint could be enabled using the environment variable setting `DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_GRPC_ENDPOINT=0.0.0.0:4317`.
```

File: content/en/tracing/setup_overview/open_standards/_index.md
Comment (jtappa) [inline content/en/tracing/setup_overview/open_standards/_index.md:339]:
```suggestion
To view all settings and environment variables supported starting Datadog Agent 7.35.0, review the `otlp_config` section in [the configuration template][26].
```

**Review by jtappa — CHANGES_REQUESTED** [review]

File: content/en/tracing/setup_overview/open_standards/_index.md
Comment (gbbr) [reply content/en/tracing/setup_overview/open_standards/_index.md:386]: Worst case scenario, even if it stays like this, I don't think the documentation will be wrong between now and then. Or, I could change it now and practically the link will work after we release.

**Review by jtappa — APPROVED** [review]

**Comment by jtappa** [comment]
Is this ok to merge, or should I add a "do not merge" label on it?

**Comment by mx-psi** [comment]
> Is this ok to merge, or should I add a "do not merge" label on it?

This should not be merged until 7.35 is released, I added the do not merge

**Review by mx-psi — APPROVED** [review]

**Comment by kayayarai** [comment]
The information in this PR has been incorporated into a bigger PR for OTel docs, #13575 . I recommend we close this one without merging. Can wait until 13575 merges if you like.

**Comment by gbbr** [comment]
@kayayarai yes I think that's fine! Feel free to close

**Pull request closed without merging**
